orchestration/templates/noop-flow.yaml

```diff
@@ -0,0 +1,52 @@
+## No-op flow for testing out argo lifecycle hooks
```

Comment:
☝️ I added this because:
1.  We don't have a way to exercise these workflows locally
2. I don't want to have to run a full e2e hca workflow to test this simple lifecycle hook.